.devcontainer/devcontainer.json

```diff
@@ -1,17 +1,32 @@
 // If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
 {
 	"name": "Go",
-	"dockerComposeFile": "docker-compose.yml",
-    "service": "docker-in-docker",
+	"dockerFile": "./Dockerfile",
 	"workspaceFolder": "/workspace",
+	"context": "..",
+	"workspaceMount": "src=${localWorkspaceFolder},dst=/workspace,type=bind",
 	"postCreateCommand": "make set-kindcluster",
-    "shutdownAction": "stopCompose",
+    "runArgs": [
```

Comment:
Last time we tried we had an issue with passing environment variables file with runArgs,  --env-file=.env didn't work for us.
We need to pass these to variables as environment variables 
```
DATABRICKS_TOKEN=xxxxxxx
DATABRICKS_HOST=https://australiaeast.azuredatabricks.net

``` 
@JasonTheDeveloper might remember it better than me but we bumped into a few issues that's why we used docker-compose

Reply:
Sorry I didn't port this over from the docker compose version, my bad. I've pushed and update with this working by using the `localWorkspaceFolder` var

`"--env-file", "${localWorkspaceFolder}/.env"`

Reply:
Gave it a quick test and seemed to mount the envs correctly into container.